src/main.h

```diff
@@ -1303,6 +1303,7 @@ class CBlockIndex
     // Gridcoin (17.6.2017 Brod) Add dpor related back-pointers
     CBlockIndex* ppCpid;  // previous block by same cpid
     CBlockIndex* ppSuper; // previous superblock
+    CSuperblock* pDesSuper; // deserialized superblock data

```

Comment:
Then keeping it in the block itself should be fine, right? I know it's just 4 or 8 bytes but for the current chain it's 37 or 74 megs and scaling :)